get_together/views/user.py

```diff
@@ -27,6 +28,26 @@ def login(request):
         return redirect('home')
     return render(request, 'get_together/users/login.html')
 
+
+def show_profile(request, user_id):
+
+    template = 'get_together/users/show_profile.html'
+
+    try:
+        user = UserProfile.objects.get(id=user_id)
+    except ObjectDoesNotExist:
+        return render(request, template, {'user': None}, status=404)
+
+    teams = Member.objects.filter(user_id=user_id)
```

Comment:
This doesn't do what you think it does. Member.user is a ForeignKey to UserProfile, not User, and this will return a set of Member records instead of Team records. You'll notice this this causes the links on the show-profile page to be incorrect.

Instead use:
`teams = user.memberships.all()`

That should give you a list of Teams connected to the UserProfile through the Member models.

Reply:
Good catch! It's fixed now.